lib/IRGen/IRGenSIL.cpp

```diff
@@ -931,22 +931,24 @@ class IRGenSILFunction :
     }
   }
 
-  // SWIFT_ENABLE_TENSORFLOW
-  // Returns the LLVM function with `funcName`. It must exist.
-  llvm::Function *findFunction(StringRef funcName, SILModule &silModule) {
```

Comment:
Removing this is great! But I think it deleted the SWIFT_ENABLE_TENSORFLOW that should've stayed here because the code below is TF.

Reply:
You are right. brought back this comment.